packages/components/src/popover/index.js

```diff
@@ -100,6 +100,13 @@ class Popover extends Component {
 		this.rafHandle = window.requestAnimationFrame( () => this.computePopoverPosition() );
 	}
 
+	/**
+	 * Calling `refresh()` will force the Popover to recalculate its size and
+	 * position. This is useful when a DOM change causes the anchor node to change
+	 * position.
+	 *
+	 * @return {void}
```

Comment:
There's a bit of a convention to not explicitly document the undefined return value. I seem to recall some discussion on it, but it should be formalized if we want to follow one way or the other.

Reply:
Can we add a lint rule one way or the other?

Reply:
Looking at this page, it might be possible by `"requireReturn": false`:

https://eslint.org/docs/rules/valid-jsdoc#options

>false if and only if the function or method has a return statement or returns a value e.g. async function (this option value does apply to constructors)

Reply:
It looks like that's what we're doing:

https://github.com/WordPress/gutenberg/blob/25f364bfabb775f5bd591a619e92d6d050fd0f9e/eslint/config.js#L155

I guess there's no way to _require no `@return {void}`_ which is a shame.

Which is the lesser of two evils? Having to type `@return {void}` all the time, or allowing inconsistency? 😛

Reply:
Or creating our own JSDoc rule!